build.gradle.kts

```diff
@@ -103,7 +103,8 @@ fun Project.configureAndroid() {
                     "InvalidPackage", // Firestore uses GRPC which makes lint mad
                     "NewerVersionAvailable", "GradleDependency", // For reproducible builds
                     "SelectableText", "SyntheticAccessor", // We almost never care about this
-                    "MissingTranslation" // TODO remove in future PR
+                    "MissingTranslation", // TODO remove in future PR
+                    "IconMissingDensityFolder"
```

Comment:
We shouldn't be ignoring this, looks like you added an image at only MDPI.

Options are:
1) Make the other densities (I can help you with a tool)
2) Use a vector asset